docs/releases/1.9.txt

```diff
@@ -416,6 +416,15 @@ Miscellaneous
   Therefore, the ``@skipIfCustomUser`` decorator is no longer needed to
   decorate tests in ``django.contrib.auth``.
 
+* In Generic Class Based Views,
+  :meth:`~django.views.generic.edit.FormMixin.get_context_data` was added to
+  :class:`~django.views.generic.edit.FormMixin` and form initialization was moved from
+  :class:`~django.views.generic.edit.ProcessFormView`'s :meth:`~django.views.generic.edit.ProcessFormView.get`
+  method to the new method. This may be backwards incompatible if you have
+  overridden the ``get_context_data`` method without using ``super()`` function
```

Comment:
get_context_data() (add parens)
"without calling super() in an overridden `get_context_data()` method."